[BUG] Growatt HV CAN comparison is always true

**Describe the bug**
```
Software/src/inverter/GROWATT-HV-CAN.cpp: In member function 'virtual void GrowattHvInverter::update_values()':
Software/src/inverter/GROWATT-HV-CAN.cpp:32:94: warning: comparison is always true due to limited range of data type [-Wtype-limits]
   32 |   if (datalayer.battery.info.number_of_cells >= 10 && datalayer.battery.info.number_of_cells <= 512) {
      |                                                       ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^~~~~~
```

**To Reproduce**
- checkout 53474e9
- run clean
- run build
- check terminal output

**Expected behavior**
No warning

**Version and settings (please complete the following information):**
- Software version: 53474e9
- Hardware used for Battery-Emulator: Stark CMR

--- a/Software/src/inverter/GROWATT-HV-CAN.cpp
+++ b/Software/src/inverter/GROWATT-HV-CAN.cpp
@@ -29,7 +29,7 @@ void GrowattHvInverter::
   // Use that when it is sane. Keep the hard-coded values as a fallback for integrations that do not populate
   // number_of_cells (e.g. some Pylon paths set it to a placeholder value).
   uint16_t total_cells = TOTAL_NUMBER_OF_CELLS;
-  if (datalayer.battery.info.number_of_cells >= 10 {
+  if (datalayer.battery.info.number_of_cells >= 10) {
     total_cells = (uint16_t)datalayer.battery.info.number_of_cells;
   }
   // If we are using a dynamic cell count, a safe default for "modules in series" is 1 unless your integration

